.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### Description
+[Describe what this change achieves]
+ 
+### Issues Resolved
+[List any issues this PR will resolve]
+ 
+### Check List
+- [ ] New functionality includes testing.
+  - [ ] All tests pass
+    - [ ] `yarn test:jest`
+    - [ ] `yarn test:jest_integration`
+    - [ ] `yarn test:ftr`
```

Comment:
Does OUI also have the same tests and commands?

Reply:
No it doesn't. Good catch!